src/accordion/stories/index.css

```diff
@@ -0,0 +1,109 @@
+.Accordion {
```

Comment:
Can we use small casing for css classes?

Reply:
I just copy pasted from the WAI-ARIA site.
Since this is just an example, I did not care about the case.

Reply:
I have changed it !!